docs/linting/troubleshooting/TSLINT.md

```diff
@@ -10,9 +10,10 @@ TSLint was a linter equivalent to ESLint that was written specifically to work d
 
 ## Migrating from TSLint to ESLint
 
-If you are looking for help in migrating from TSLint to ESLint, see [`tslint-to-eslint-config`](https://github.com/typescript-eslint/tslint-to-eslint-config).
+The standard tool to convert a TSLint configuration to the equivalent ESLint configuration is [`tslint-to-eslint-config`](https://github.com/typescript-eslint/tslint-to-eslint-config).
+If you are still using TSLint, we strongly recommend you use this tool to switch.
 
-You can look at [`the plugin ROADMAP.md`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/ROADMAP.md) for an up to date overview of how TSLint rules compare to the ones in this package.
+You can look at [the `@typescript-eslint/eslint-plugin` `ROADMAP.md`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/ROADMAP.md) for an up to date overview of how TSLint rules compare to the ones in this package.
```

Comment:
one thing I hate is that we call this "roadmap" - it implies we're planning on implementing things. Ideally we should rename the file, but in the short term perhaps we should just stop referring to it as "ROADMAP.md"?

```suggestion
You can look at [the alternatives list](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/ROADMAP.md) for an up to date overview of how TSLint rules compare to the ones in this package.
```

Reply:
I can just move the file in this PR and leave a redirect notice. 👍